src/vendorcode/dasharo/include/dasharo/options.h

```diff
@@ -245,4 +248,21 @@ uint16_t dasharo_apu_watchdog_timeout(void);
  */
 bool dasharo_apu_cpu_boost_enabled(void);
 
+/* Looks up Dasharo/"CpuThrottlingThreshold" variable to return the CPU
+ * throttling threshold temperature.
+ *
+ * Result:
+ *  returns uint8 - cpu_throttling_threshold
+ */
+uint8_t get_cpu_throttling_threshold(void);
+
+/* Looks up Dasharo/"CpuMaxTemperaure" variable to return the maximum CPU
+ * temperature (TjMax).
+ *
+ * Result:
+ *  returns uint8 - tjmax
+ */
+uint8_t get_cpu_max_temperature(void);
```

Comment:
Hmm, does EDK2 ever modify this variable? If not, you can just use CONFIG_CPU_MAX_TEMPERATURE directly instead of adding this function